nodejs/week1/src/backend/routes/meals.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const app = express.Router();
+const mealsRouter = require("../data/meals.json");
+const reviewsRouter = require("../data/reviews.json");
+
+//Respond with the json for all the meals
+app.get("/meals", (req, res) => {
+  mealsRouter.forEach((meal) => {
```

Comment:
You don't have to use parenthesis, if there is one attribute.
`forEach((meal) => `could be` forEach(meal => `